tools/wake-format/wake-format.cpp

```diff
@@ -167,7 +174,9 @@ int main(int argc, char **argv) {
     exit(EXIT_FAILURE);
   }
 
-  wcl::xoshiro_256 rng(wcl::xoshiro_256::get_rng_seed());
+  auto seed = (no_rng) ? std::tuple<uint64_t, uint64_t, uint64_t, uint64_t>({0, 0, 0, 0})
```

Comment:
Oh interesting we need this because the tmp file is a part of the error message and we need to be deterministic but we *also* need running two of the same wake-formats to not conflict....interesting....Yeah I don't see a good way around this.